docs/concepts/environments.md

```diff
@@ -30,16 +30,17 @@ and are commonly used for self-contained projects or different subprojects
 within a monorepo.
 1. **Centrally managed environment**: An environment stored remotely on
```

Comment:
I wouldn't change any of the words, but I think with this amount of text we need to break it up a bit. The way this page is set up, it's meant to be a high level overview of these env uses, so let's make it so the reader can quickly see the definition first, and the other info in related chunks, something like this:

```
1. **Path environment**: An environment stored in a local directory. 
  - This environment is self contained in the `.flox` directory and can be
reproduced by sharing the directory in version control or some other file
sharing mechanism.
  - Path environments are created with [`flox init`][flox_init],
referred to with the `--dir/-d` option on most CLI commands,
and are commonly used for self-contained projects or different subprojects
within a monorepo.

2. **Centrally managed environment**: An environment stored remotely on
[FloxHub][floxhub_concept].

  - Centrally managed environments are created by running [`flox push`][flox_push]
on a path environment. You can connect a new project directory with an existing centrally managed environment with [`flox pull ...`][flox_pull] or
you can activate the environment directly with [`flox activate --remote ...`][flox_activate] for
instant use.
  - Centrally managed environments enable multiple projects or systems to consume a
shared environment that is versioned with [generations][generation_concept].
They are commonly used as base environments for projects of similar tech stacks,
for reproducing issues on specific systems, or to quickly share tools.
  - To disconnect a centrally managed environment from FloxHub, run [`flox pull --copy`][flox_pull] instead of `flox pull`.
This will turn the environment back into a path environment.
```

Reply:
thank you, i've applied your changes